js-core-2/lesson7.md

```diff
@@ -32,59 +32,60 @@ When we are you solving a problem, one of your main decisions as a developer is
 
```

Comment:
These were already in the notes - I've added the wallet exercises underneath but I've left these in the lessons as a sort of recap. I'll look into rewriting them so they're more in-depth with how you can actually use objects.